server/index.js

```diff
@@ -5,13 +5,25 @@ const port = parseInt(process.env.PORT, 10) || 3000;
 const dev = process.env.NODE_ENV !== 'production';
 const app = next({ dev });
 const handle = app.getRequestHandler();
+const multer = require('multer');
+const uuidv4 = require('uuid/v4');
 
 const content = require('./content');
 const contentType = require('./content-type');
+const currentDate = `${new Date().getMonth() +
```

Comment:
this const should cannot be in the global scope as it’s going to be dynamic per request. once it’s defined in global scope it’ll only refresh on full server restart